src/services/api.js

```diff
@@ -27,12 +30,23 @@ export const login = (userType, email, password) => {
     })
 }
 
+export const getVaccinationSlots = (page = 1, startDate = null, endDate = null, onlyReserved = null) => {
+    return api.get(`/doctor/vaccination-slots?page=${page}
+        ${startDate ? '&startDate=' + startDate : ''}
+        ${endDate ? '&endDate=' + endDate : ''}
+        ${onlyReserved ? '&onlyReserved=' + onlyReserved : ''}`);
+}
+
```

Comment:
Wszystkie sloty pokazują się jako wolne, pomimo tego, że są zarezerwowane.
![obraz](https://user-images.githubusercontent.com/57873253/160903335-84bf78d0-6c2b-4d52-a4cc-35937e9042b8.png)
![obraz](https://user-images.githubusercontent.com/57873253/160903411-f5da08f8-d5ff-45cf-b645-a780366d0e51.png)

Reply:
To dlatego, że mimo bycia zarezerwowanym mają pusty obiekt "vaccination", co jest niezgodne z api IIRC